python-app/api-diff/action.yaml

```diff
@@ -0,0 +1,98 @@
+name: OpenApi Diff
+description: Generate and comment OpenAPI Diff
+
+inputs:
+  base_ref:
+    description: "Base branch for the diff"
+    required: true
+    default: "main"
```

Comment:
Is base ref required as there's a default value?
We want not to have it mandatory if we can always diff with main ?